GMA.DataAccess/Resources/TablesScript.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS Tag (
+TagId INTEGER PRIMARY KEY AUTOINCREMENT, 
+Name NVARCHAR(50) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS Owner (
+OwnerId INTEGER PRIMARY KEY AUTOINCREMENT,
+Name NVARCHAR(200) NOT NULL,
+ContactDetails NVARCHAR(500)
+);
+
+CREATE TABLE IF NOT EXISTS Server(
+ServerId INTEGER PRIMARY KEY AUTOINCREMENT,
+Name NVARCHAR(200) NOT NULL,
+Description NVARCHAR(500) NOT NULL,
+ServerLink NVARCHAR(300) NOT NULL,
+Port INTEGER NOT NULL,
+OwnerId INTEGER NOT NULL,
```

Comment:
We don't need to have OwnerId not null, it can be nullable

Reply:
fixed